### PR TITLE
Use correct storage to store results

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: workflows/templates/advise-template.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ spec:
             archive:
               none: {}
             s3:
-              key: "argo/adviser/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+              key: "data/thoth/thoth-psi-stage/adviser/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
```
Comment (fridex) [inline workflows/templates/advise-template.yaml]: Can this be obtained from `thoth` configmap not to hardcode it to the template?

Comment (pacospace) [reply workflows/templates/advise-template.yaml]: there is way, @CermakM suggested me one solution, I will implement it in another PR :)

Comment (fridex) [reply workflows/templates/advise-template.yaml]: So I guess you want this one to be merged, or will it replace this one? :)

Comment (pacospace) [reply workflows/templates/advise-template.yaml]: This can be merged meanwhile :) I need to test the new solution.

Comment (fridex) [reply workflows/templates/advise-template.yaml]: :crossed_fingers:

**Pull request merged**